nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -62,12 +62,23 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
         matrixWriter = null
     }
 
+    private fun readCharacteristics() {
+        readBatteryPercentage()
+        readFirmwareVersion()
+    }
+
```

Comment:
Please call both read methods directly from `onCharacteristicChanged`, no need for that wrapper method